ui/app/components/app/connected-accounts-list/connected-accounts-list-options-item/connected-accounts-list-options-item.component.js

```diff
@@ -0,0 +1,27 @@
+import classnames from 'classnames'
+import PropTypes from 'prop-types'
+import React, { PureComponent } from 'react'
+
+export default class ConnectedAccountsListOptionsItem extends PureComponent {
+  static propTypes = {
+    iconClassNames: PropTypes.string,
+    children: PropTypes.node.isRequired,
+    onClick: PropTypes.func,
+  }
+
+  static defaultProps = {
+    iconClassNames: undefined,
+    onClick: undefined,
+  }
+
+  render () {
+    const { children, iconClassNames, onClick } = this.props
+
+    return (
+      <div className="connected-accounts-options__row" onClick={onClick}>
```

Comment:
Could this be made into a `button` instead? It would be better to use a native browser element that can be clicked, to inherit the accessibility properties of that element.

Reply:
This is nested inside of a button already, so no. I can add a role to it.